src/inc/CombFilter.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+
+//IIR class CCombFilterBase;
+#include <math.h>
+#include <algorithm> 
+#include <complex>
+using namespace std;
```

Comment:
Don't leave things like this in your code if you're not using any STLs. Also using namespace std is not recommended anyways.